recipes/rbc/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "rbc" %}
+{% set version = "0.1.0.dev1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xnd-project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: c9566d7e820e684dbcac01baa11a7fde5048ecbfcea0d0d929ebfdea764a542a
+
+build:
+  number: 0
+  skip: True  # [py<35 or win]
+  script: "{{ PYTHON }} setup.py install"
```

Comment:
See https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip

Reply:
Fixed, now using pip to install the package.